lib/feeder/api.ex

```diff
@@ -1,58 +1,55 @@
 defmodule Feeder.Api do
+  use Memoize
 
-  @per 100
+  @per 200
   @max_pages 4
   @ever "1970-01-01"
 
-  def podcasts(), do: get_items("/api/v1/podcasts", %{since: @ever, per: @per})
-  def podcasts(since), do: get_items("/api/v1/podcasts", %{since: format(since), per: @per})
-
-  def episodes(), do: get_items("/api/v1/episodes", %{since: @ever, per: @per})
-  def episodes(pid) when is_integer(pid), do: get_items("/api/v1/podcasts/#{pid}/episodes", %{since: @ever, per: @per})
-  def episodes(since), do: get_items("/api/v1/episodes", %{since: format(since), per: @per})
-  def episodes(pid, since), do: get_items("/api/v1/podcasts/#{pid}/episodes", %{since: format(since), per: @per})
-
-  def get_items(path, params, depth \\ 1) do
-    case HTTPoison.get(url(path), [], params: params) do
-      {:ok, %{status_code: 200, body: body}} ->
-        case Poison.decode(body) do
-          {:ok, json} -> get_item_pages(json, depth)
-          _any -> {:error, "invalid json from #{url(path)}"}
-        end
-      {:ok, %{status_code: code}} -> {:error, "got #{code} from #{url(path)}"}
-      {:error, %{reason: reason}} -> {:error, reason}
-    end
+  defmemo root(), expires_in: :timer.seconds(300) do
+    Env.get(:feeder_host)
+    |> PrxAccess.root(
+      account: "*",
```

Comment:
This is the ID "wildcard auth".  Your client id/secret must be for the id `admin` user (8).  Otherwise you'll get an unauthorized error.